plugins/textserver/CMakeLists.txt

```diff
@@ -3,12 +3,7 @@
 ###########################################
 add_library(textserver SHARED textserver.cpp textserver.h plugindefs.h)
 
-if( MSVC )
-  target_link_libraries(textserver libopenrave imm32 winmm ws2_32)
-else()
-  target_link_libraries(textserver libopenrave)
-endif()
-target_link_libraries(textserver PRIVATE boost_assertion_failed)
+target_link_libraries(textserver PUBLIC libopenrave $<$<CXX_COMPILER_ID:MSVC>:imm32> $<$<CXX_COMPILER_ID:MSVC>:winmm> $<$<CXX_COMPILER_ID:MSVC>:ws2_32> PRIVATE boost_assertion_failed)
```

Comment:
@undisputed-seraphim congrats on making the one-liner harder to read, please revert back unless you have a good reason for doing it this way.

Reply:
I agree, and reverted.